python/hidet/ir/primitives/cuda/mma.py

```diff
@@ -94,6 +94,25 @@ def __str__(self):
         return self.inst_name()
 
 
+class MmaConfigInt(MmaConfig):
+    def inst_name(self) -> str:
+        return 'mma.sync.aligned.m{}n{}k{}.row.col.satfinite.{}.{}.{}.{}'.format(
+            self.m, self.n, self.k, self.output_dtype, self.input_dtype, self.input_dtype, self.output_dtype
+        )
+
+    @staticmethod
+    def m8n8k16_i8_i32():
+        return mma_configs['m8n8k16_i8_i32']
+
+    @staticmethod
+    def m16n8k16_i8_i32():
+        return mma_configs['m16n8k16_i8_i32']
+
+    @staticmethod
+    def m16n8k32_i8_i32():
+        return mma_configs['m16n8k32_i8_i32']
+
+
```

Comment:
Okay, if you have this already implemented, I can delete this. The only thing is the .satfinite identifier, which I am not sure is even necessary for u8/i8 and u4/i4. Since 32 * 255^2 < 2^32 - 1.

Reply:
You do not need to delete this. Just implement inside the original `MmaConfig` instead of creating a new class `MmaConfigInt` (like what I did in the unmerged commit). I will not working on that branch in the recent 1~2 months.